cluster/migrations/000002_url-tokens.up.sql

```diff
@@ -0,0 +1,43 @@
+-- This file is part of MinIO Kubernetes Cloud
+-- Copyright (c) 2019 MinIO, Inc.
+--
+-- This program is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU Affero General Public License as published by
+-- the Free Software Foundation, either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Affero General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
+-- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+create table url_tokens
+(
+    id          uuid
+        constraint url_tokens_pk
+            primary key,
+    tenant_id uuid not null
+        constraint url_tokens_tenant_id_fk
+            references tenants
+            on delete cascade,
+    user_id uuid not null,
+    expiration  timestamptz,
+    used_for    varchar(256),
```

Comment:
should we better use a type for `used_for`? so that we let the db validate the use, else we need to validate by strings every time.

Reply:
Yes, we could introduce an enumerated type on postgres, or all together use a golang type